client/material/popover.jsx

```diff
@@ -193,7 +198,8 @@ export default class Popover extends React.Component {
     const rect = {
       top: anchorElement.top + anchorOffsetVertical,
       right: anchorElement.right + anchorOffsetHorizontal,
-      bottom: anchorElement.bottom + anchorOffsetVertical,
+      // bottom: anchorElement.bottom + anchorOffsetVertical,
+      bottom: 600,
```

Comment:
would *really* recommend using something that breaks the linter when you do stuff like this, e.g.
`bottom: 600, // FIXME`